packages/amplication-server/src/core/serviceSettings/serviceSettings.service.ts

```diff
@@ -161,11 +162,13 @@ export class ServiceSettingsService {
     user: User,
     serviceSettings: ServiceSettingsUpdateInput = null
   ): Promise<ServiceSettings> {
-    const settings = DEFAULT_SERVICE_SETTINGS;
+    const defaultSettings = DEFAULT_SERVICE_SETTINGS;
 
-    if (serviceSettings)
-      // for backwards compatibility, we need to update the service settings with the new field (generateServer)
-      this.updateServiceGenerationSettings(settings, serviceSettings);
+    // if (serviceSettings)
```

Comment:
Looks good, I think we can remove the commented out part ( I assume you left it for the review to be easier)